pkg/util/aci.go

```diff
@@ -31,6 +43,22 @@ func NewACI(name string) (*os.File, error) {
 
 	tw := tar.NewWriter(tf)
 	aw := aci.NewImageWriter(im, tw)
+
+	for _, entry := range entries {
+		// Add default mode
+		if entry.Header.Mode == 0 {
+			if entry.Header.Typeflag == tar.TypeDir {
+				entry.Header.Mode = 0755
+			} else {
+				entry.Header.Mode = 0644
+			}
+		}
+		sr := strings.NewReader(entry.Contents)
+		if err := aw.AddFile("", entry.Header, sr); err != nil {
```

Comment:
er, hmm, AddFile is clearly in need of an update since it does nothing with the first arg..